lib/plugins/aws/deploy/lib/createStack.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
   create() {
     this.serverless.cli.log('Creating Stack...');
     const stackName = `${this.serverless.service.service}-${this.options.stage}`;
+    const stackTags = this.serverless.service.provider.stackTags || { STAGE: this.options.stage };
```

Comment:
Will sort that today or tomorrow morning. FYI Ad. 2 It intentionally enables users to replace the default `STAGE` tag for two reasons:
1. Use lower case 'stage' if this is already applied aws tagging convention
2. Use different tag instead of 'STAGE' - for instance we have been using 'environment' for years.
No problems with any these though, will still work for me perfectly fine.